index.html

```diff
@@ -7,7 +7,6 @@
     <title>BC Business Registry</title>
     <link rel="icon" href="dummy-not-used-do-not-remove">
     <link rel="icon" href="/favicon.png">
-    <link rel="preload" href="/fonts/materialdesignicons-webfont.927457ed.woff2" as="font" type="font/woff2" crossorigin>
```

Comment:
I deleted this because it was displaying a console error, because the path (/fonts) doesn't exist (in /public). Works fine (notice the icons on the page):

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/c89b0728-57a5-467b-9803-07f549e38319)

Also, neither [Create UI](https://github.com/bcgov/business-create-ui/blob/main/index.html) nor [Edit UI](https://github.com/bcgov/business-edit-ui/blob/main/index.html) use this, so I think we're OK.